docs/en/ingest-management/troubleshooting.asciidoc

```diff
@@ -4,12 +4,113 @@
+
+[[enrolled-agent-not-showing-up]]
+=== I ran the Agent enrollment step but I don’t see the Agent in the UI
+If Elastic Agent was successfully enrolled, but doesn't show up in the Fleet list a common problem is that it was not 'started'.  Starting the Agent is a separate step which can be done as follows as the below.
+
+On linux & macos hosts:
+./elastic-agent run
+
+On Windows hosts:
+elastic-agent.exe run
+
+[[where-are-the-agent-logs]]
+=== I need to find the logs when the Agent starts up
+
+When started successfully, the logs are in the folder where the Agent was started, in a sub-directory 'data/logs/metricbeat'
+If that path of logs does not exist, it indicates Agent did not successfully run metricbeat, which is a higher level problem to triage.
+
+[[what-is-my-agent-config]]
+=== I need to know the configuration used in the running Agent.
+
+You can inspect the files in the folder from where you ran the Agent.  The elastic-agent.yml file will have an uncommented citation if it is in 'Fleet' mode:
+Management: mode: "fleet"
+
+The action_store.yml has the whole configuration, unencrypted.  Look for 'outputs:hosts' section with elasticsearch location.  This file will also have the current versions of the packages in use by the current configuration.  The version of the Agent is in the download folder / zip.
+
+[[where-is-the-data-agent-is-sending]]
+=== I'm not sure {ingest-manager} app is showing the data my Agent is sending.
+
+I think I have Agent set up and running and there are no errors on the Agent logs and I still don't see data in Kibana.  The recommended way to assess this is to go straight to Elasticsearch data queries, or maybe just use Kibana Discover app.  Look for any data in the index your Agent is sending to (metrics-* for example) and see if the name.host of your host machine shows.  If not, then perhaps the data is getting trapped / blocked in the network or isn't sent off of the Agent host correctly in the first place due to some Firewall or security reason.  While it should be redundant, it is all the same potentially helpful to install the stand-alone Metricbeat (see the Elastic docs for information) to see if that works to send data into ES, which would indicate a bug or problem in the Agent side.
+
+[[i-deleted-my-agent]]
+=== I deleted my agent in the Fleet {ingest-manager} app so now what?
+
+Its ok, be cool: the data is still in Elasticsearch.  But to get the Agent back in the UI, you will need to stop, then re-enroll the Agent on the host and run it again.
+
+[[i-need-to-stop-agent]]
+=== I need to stop the Agent and all processes running on my host
```

Comment:
This doesn't seem like a troubleshooting topic. Seems like something that belongs somewhere else in the docs. If you want, I can more it over and put it in the correct style for the docs.

Reply:
This is fair, but for the sake of helping users triage and use material, I'd like to leave it here - if it also benefits them to be elsewhere we can add it where it needs to be?  We have a whole slew of content about using the product that is surely to be worked on (I haven't looked at any of it) and this is just one portion that impacts a user's personal set of host systems and it seems helpful to leave (even if intentionally overlapping).